packages/catalog-model/src/kinds/TemplateEntityV1beta2.test.ts

```diff
@@ -122,16 +123,29 @@ describe('templateEntityV1beta2Validator', () => {
     delete (entity as any).spec.steps[0].action;
     await expect(validator.check(entity)).rejects.toThrow(/action/);
   });
+
   it('accepts missing owner', async () => {
     delete (entity as any).spec.owner;
     await expect(validator.check(entity)).resolves.toBe(true);
   });
+
   it('rejects empty owner', async () => {
     (entity as any).spec.owner = '';
     await expect(validator.check(entity)).rejects.toThrow(/owner/);
   });
+
   it('rejects wrong type owner', async () => {
     (entity as any).spec.owner = 5;
     await expect(validator.check(entity)).rejects.toThrow(/owner/);
   });
+
+  it('accepts missing if', async () => {
+    delete (entity as any).spec.if;
+    await expect(validator.check(entity)).resolves.toBe(true);
+  });
+
+  it('accepts boolean in if', async () => {
+    (entity as any).spec.if = true;
+    await expect(validator.check(entity)).resolves.toBe(true);
+  });
```

Comment:
add `rejects wrong type if` (like a number or so) and `accepts empty if` (the empty string) I'd say, since those are common things for the schema to deal with

Reply:
Tests got updated. Also fixed `accepts boolean in if` to work properly 🤦